canton/BUILD.bazel

```diff
@@ -834,11 +835,17 @@ genrule(
 dar_to_scala(
     name = "community_participant_dar_scala",
     srcs = [
-        ":admin-workflows-dar",
-        ":admin-workflows-with-vacuuming-dar",
-    ],
+        ":community_participant_admin-workflows_dar",
+        ":community_participant_admin-workflows-with-vacuuming_dar"
+        ],
     package_prefix = "com.digitalasset.canton.participant.admin.workflows",
-    srcjar_out = "dar.srcjar",
+    srcjar_out = "community_participant_admin-workflow_scala.srcjar",
```

Comment:
Is this guy actually still used?

Reply:
Yes in `canton/community/participant/src/main/scala/com/digitalasset/canton/participant/admin/workflows/PackageID.scala`